CRM/ManualDirectDebit/Form/SetUp.php

```diff
@@ -70,6 +72,68 @@ public function buildQuickForm() {
 
   public function postProcess() {
     parent::postProcess();
+
+    $values = $this->exportValues();
+
+    $contributionRecur = civicrm_api3('Contribution', 'getsingle', [
+      'id' => $values['contribution_id'],
+      'api.ContributionRecur.get' => [],
+    ])['api.ContributionRecur.get']['values'][0];
+
+    $cycleDay = $this->getCycleDay($values['payment_dates'], $contributionRecur['frequency_unit']);
+
+    $test = civicrm_api3('ContributionRecur', 'create', [
```

Comment:
`$test` is a bit generic. Plus it doesn't seem to be used anywhere else in the code, so I guess we can just remove it.

Reply:
@MiyaNoctem  good catch, I was using it for debugging the result. Will remove.